### ISSUE TITLE
props supports.css

### ISSUE THREAD

**Issue by woodcox** [mayank99/open-props-scss#5]
Hi, it looks like you could include the open props package called supports.css. I presume Adam added this one fairly recently as it’s missing from your repo.

Would you be open to a pull request for this?

**Comment by woodcox** [comment]
Having looked further at [open props GitHub repo](https://github.com/argyleink/open-props/tree/main/src) I’ve realised that it’s not in there index.css or index.js file either. So I understand why you didn’t include it.

**Comment by mayank99** [comment]
I'm still open to adding it; looks quite handy. However, the approach will need to differ. Instead of media queries or custom properties, we'd want to make them mixins that use `@supports` and [`@content`](https://sass-lang.com/documentation/at-rules/mixin#content-blocks).

For example:
```scss
@mixin safari-only {
  @supports (-apple-pay-button-style: inherit) {
    @content;
  }
}
```

**Comment by woodcox** [comment]
I've realised there is no `props.supports.js` file in the [src of open-props git repo](https://github.com/argyleink/open-props/tree/main/src), so this will need to wait until one is created.

**Comment by mayank99** [comment]
we can always read the contents of the css file and extract the values from there

**Comment by woodcox** [comment]
Yeah, I've opened an [issue on open-props](https://github.com/argyleink/open-props/issues/380). I'll give it a bit of time to see if it was an oversight first.
